src/index.tsx

```diff
@@ -48,6 +53,109 @@ function Updaters() {
     )
 }
 
+const Input = styled.input<{ error?: boolean }>`
+    border-radius: 10px;
+    width: 100%;
+`
+const Img = styled.img`
+    height: 100px;
+`
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+    padding: 10%;
+    justify-content: center;
+    align-items: center;
+`
+const Text = styled.div<{ fontSize?: number }>`
+    font-style: normal;
+    font-size: ${({ fontSize }) => (fontSize ? fontSize + 'px' : '15px')};
+    line-height: 166%;
+    letter-spacing: 0.35px;
+    color: #787878;
+    text-align: center;
+
+    @media screen and (max-height: 720px) {
+        font-size: 12px;
+        margin-top: 30px;
+    }
+`
+const Link = styled.a`
+    color: blue;
+`
+
+function CustomApp() {
+    const [auth, setAuth] = useState(false)
+    const [cookies, setCookie, removeCookie] = useCookies(['pwd'])
+    const PASSWORD = 'gdbetatest'
+    const PASSWORD_HASH = '$2a$10$V9DPoPvZtRpg9t23wzl5c.jYYyG5VJdJx/pvBJy61WmN/01rkSfSm'
+
+    useEffect(() => {
+        if (cookies.pwd === PASSWORD_HASH) setAuth(true)
```

Comment:
also read possible initial password from url. ie
website.tld/?pass=gdbetatest

Reply:
Hey @sirpy,

I have implemented this screen where you'll need to input the password manually as discussed with @MayurAdaptiveCode. 

Can you elaborate more on the url based implementation?